modules/utility/info.rb

```diff
@@ -1,152 +1,181 @@
+            inline: true
+          )
+          embed.add_field(
+            name: 'Account Created:',
+            value: "#{user.creation_time.getutc.asctime} UTC",
+            inline: true
+          )
+          embed.add_field(
+            name: 'Joined Server:',
+            value: ignoreserver ? '[N/A]' : "#{member.joined_at.getutc.asctime} UTC",
+            inline: true
+          )
+          embed.add_field(
+            name: 'Status',
+            value: user.status.capitalize
+          )
         end
       },
-      triggers: %w(info profile),
+      triggers: %w[info profile]
     )
 
-    $cbot.add_command(:ping,
-      code: proc {|event, _|
-        return_message = event.respond('Pinging..!')
-        ping = (return_message.id - event.message.id) >> 22
-	      choose = %w(i o e u y a)
-        return_message.edit("P#{choose.sample}ng! (`#{ping}ms`)")
-       },
-      triggers: %w(ping pong peng pung pyng pang 🅱ing)
+    YuukiBot.crb.add_command(
+      :ping,
+      code: proc { |event, _|
+              return_message = event.respond('Pinging..!')
+              ping = (return_message.id - event.message.id) >> 22
+              choose = %w[i o e u y a]
+              return_message.edit("P#{choose.sample}ng! (`#{ping}ms`)")
+            },
+      triggers: %w[ping pong peng pung pyng pang 🅱ing]
     )
 
-
-    def self.ms_to_time(ms)
-      time = ms / 1000
+    def self.ms_to_time(milliseconds)
+      time = milliseconds / 1000
       seconds = time % 60
-      time = time / 60
+      time /= 60
       minutes = time % 60
-      time = time / 60
+      time /= 60
       hours = time % 24
-      time = time / 24
+      time /= 24
       days = time
 
       seconds = seconds.floor
       minutes = minutes.floor
       hours = hours.floor
       days = days.floor
 
-      seconds = (seconds.to_s.length == 1) ?  "0" + seconds.to_s : seconds.to_s
-      minutes = (minutes.to_s.length == 1) ?  "0" + minutes.to_s : minutes.to_s
-      hours = (hours.to_s.length == 1) ?  "0" + hours.to_s : hours.to_s
+      seconds = seconds.to_s.length == 1 ?  '0' + seconds.to_s : seconds.to_s
+      minutes = minutes.to_s.length == 1 ?  '0' + minutes.to_s : minutes.to_s
+      hours = hours.to_s.length == 1 ? '0' + hours.to_s : hours.to_s
 
       "#{days} day#{'s' unless days == 1}, #{hours}:#{minutes}:#{seconds}"
     end
 
-    $cbot.add_command(:uptime,
-    code: proc do |event, _|
-      uptimems = (Time.now - $launch_time) * 1000
+    YuukiBot.crb.add_command(
+      :uptime,
+      code: proc do |event, _|
+        uptime_ms = (Time.now - YuukiBot.launch_time) * 1000
 
-      event.respond("I was launched on `#{$launch_time.asctime} UTC`\nThis means I have been online for `#{self.ms_to_time(uptimems)}` (`#{uptimems.floor}ms`)")
-    end
+        event.respond("I was launched on `#{YuukiBot.launch_time.asctime} UTC`\n" \
+          "This means I have been online for `#{ms_to_time(uptime_ms)}` (`#{uptime_ms.floor}ms`)")
```

Comment:
You haven't been "online" for the whole time you're running.
```suggestion
          "This means I have been running for `#{ms_to_time(uptime_ms)}` (`#{uptime_ms.floor}ms`)")
```